lab/05-extend-aks-with-aci.md

```diff
@@ -0,0 +1,113 @@
+
+Install the ACI to Kubernetes connector using the az CLI:
+```
+az aks install-connector -n ignite18 -g ignite18
+```
+
+**Output:**
+```
+Merged "ignite18" as current context in /tmp/tmpbgwvxtlp
+Deploying the ACI connector for 'Linux' using Helm
+NAME:   aci-connector-linux-eastus
+LAST DEPLOYED: Thu Sep 20 17:59:05 2018NAMESPACE: defaultSTATUS: DEPLOYED
+
+RESOURCES:
+==> v1/Secret
+NAME                                                TYPE    DATA  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  Opaque  3     1s
+==> v1/ServiceAccount
+NAME                                                SECRETS  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1        1s
+
+==> v1beta1/ClusterRoleBinding
+NAME                                                AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1s
+
+==> v1beta1/Deployment
+NAME                                                DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1        1        1           0          1s
+
+==> v1/Pod(related)
+NAME                                                             READY  STATUS             RESTARTS  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks-6c86769mzrt2  0/1    ContainerCreating  0         1s
+
+
+NOTES:
+The virtual kubelet is getting deployed on your cluster.
+
+To verify that virtual kubelet has started, run:
+
+  kubectl --namespace=default get pods -l "app=aci-connector-linux-eastus-virtual-kubelet-for-aks"
+
+Note:
+TLS key pair not provided for VK HTTP listener. A key pair was generated for you. This generated key pair is not suitable for production use.
+```
+
+The ACI connector automatically registers with the Kubernetes cluster as a virtual virtual. This virtual node can run Pods, just like the VM-based examples we've previously run.
+
+```
+kubectl get node
+```
+
+**Output:**
+```
+NAME                                         STATUS    ROLES     AGE       VERSION
+aks-nodepool1-25718494-0                     Ready     agent     1d        v1.11.2
+aks-nodepool1-25718494-1                     Ready     agent     1d        v1.11.2
+virtual-kubelet   Ready     agent     1m        v1.11.2
+```
+
+Let's deploy an application that schedules to our virtual node:
+
+```
+kubectl apply -f manifests/aci-helloworld.yaml
+kubectl get po -l app=aci-helloworld
+```
+
+**Output:**
+```
+NAME                              READY     STATUS    RESTARTS   AGE
+aci-helloworld-57bf88c4c6-rffpb   0/1       Pending   0          30s
+```
+
+This pod is launching an Azure Container Instance, on-demand, without having to manage underlying compute. Once the Pod transitions to a `Running` state we can connect directly to that Pod via its public IP address:
+
+```
```

Comment:
here you use the selector to find the app. but that hasn't been introduced, I don't think. I think it's worth a passing mention, otherwise newbies might be confused about how that command worked.